nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -198,25 +221,26 @@ def handoff_items_phase(self):
         #        if self.move_arm_task.witihin_workspace(bin_goal_pt): break
         #        rospy.logerr(f"Closer point still outside of workspace, trying even closer point {bin_goal_pt}")
         # rospy.loginfo(f"node returns {pose_result}")
-        # if unable to find patient pose place bin at predetermined position
-        # could also abort and go home instead but this decision complexity
-        # is likely not within current scope
-        # show arm movement and get input to start
+        status, pose_result = self.estimate_pose_task.execute("body")
+        bin_goal_pt = pose_result.bin_goal.point
+        rospy.loginfo(f"node returns {pose_result}")
+
         task_response = self.send_interface_request_task.execute(
             RobotStatus.BEDSIDE_DELIVER
         )
 
         # extend arm
         rospy.loginfo("Nightingale Mission Planner extending arm for handoff")
 
-        # UNCOMMENT FOR POSE GOAL
-        # if status != TaskCodes.SUCCESS:
-        #    rospy.logwarn("UNABLE TO FIND POSE. FALLING BACK TO SAFE HANDOFF POSITION")
-        #    status = self.move_arm_task.extend_restock()
-        # else:
-        #     status = self.move_arm_task.extend_handoff(bin_goal_pt)
-        # if status != TaskCodes.SUCCESS:
-        if self.move_arm_task.extend_restock() != TaskCodes.SUCCESS:
+        if status != TaskCodes.SUCCESS or not self.move_arm_task.within_workspace(
+            bin_goal_pt
+        ):
+            rospy.logwarn("UNABLE TO FIND POSE. FALLING BACK TO SAFE HANDOFF POSITION")
+            status = self.move_arm_task.extend_restock()
```

Comment:
We should probably redefine this hardcoded extend even though we should never get to it. Maybe instead create a new default extend which just lifts the box up?